### ISSUE TITLE
zilien reader EC data plot error - 'experiment_number' is not in list

### ISSUE THREAD

**Issue by tugce-y** [ixdat/ixdat#107]
**When I try to read the data, I am getting an error message:** 
_" ...\Anaconda3\lib\site-packages\ixdat\readers\zilien.py", line 344, in _biologic_dataset_part
    exp_nums = data_columns_split[:count, column_headers.index("experiment_number")]
ValueError: 'experiment_number' is not in list
""_
**The code that I am using is:**
```
df ="2023-02-16 11_47_19 IrO2_Ti_Cl.tsv"
ms = Measurement.read(df, reader="zilien")
```
If I indicate the technique="MS", I can plot the MS data with no problem. 

There is actually no "experiment_number" in the TSV file, but I can plot my old data (with an older Zilien version) with no "experiment_number" as well without a problem.  

- zilien version: 2.6.0
- ixdat version: 0.2.5 

I can provide the data privately.

**Comment by matenestor** [comment]
Yes, I have a feeling that this will be reoccurring problem. It happened, because Zilien wasn't able to find MPT files and thus couldn't integrate them into the TSV file. It can happen when you
- don't activate MPT files generation during a measurement
- when the MPT files are being generated in a different location than where the Zilien's TSV file is -- that can happen when you load EC-lab settings from a EC-lab experiment done in past.

Your data is still there, it is **not** lost. You will just need to load the MPT files separately as an Ixdat's `Measurement` and connect them with the TSV `Measurement`.

```python
tsv = Measurement.read("zilien_dataset.tsv", technique="MS")
mpt = Measurement.read_set("directory_with_mpt_files", suffix="mpt")

combined = tsv + mpt
combined.plot_measurement()
plt.show()
```

This should show you both MS and EC plots in one figure.

**However**, there is a bug that makes loading the MPT files impossible. You can get an exception
`BiologicMPTReader(file.mpt) did not find any data for t_str='time/s'. This reader only works for files with a 'time/s' column`
even though there are `time/s` columns. I am describing this problem more in here #108.

**Comment by ScottSoren** [comment]
I think this should be resolved with the recent release Zilien 2.6.0 ? 
Will that need an update in the zilien reader @matenestor ?

**Comment by matenestor** [comment]
Yes, it is a new feature in Zilien 2.6.0. If you measured with an older version then you need to specify `technique="MS"` when reading a dataset file.
The Zilien reader update is already done :slightly_smiling_face: It can read a TSV file which has properly integrated MPT data

**Comment by ScottSoren** [comment]
@tugce-y and @matenestor , this may interest you: https://github.com/ixdat/ixdat/pull/134
It will soon be possible to just read the .mpr files when something like this is wrong with the .mpt files :) . Feel free to comment on that PR.

**Comment by matenestor** [comment]
Fixed in #159
